includes/Plugin.php

```diff
@@ -219,31 +219,70 @@ public static function checkPluginCompatabilityAndPrintErrorMessages(): bool
      */
     public function validateSelectedFulfillmentStates(): void
     {
-        $fulfillmentState = $this->settingsRepository->getFulfillmentState();
-        $fulfilledState = $this->settingsRepository->getFulfilledState();
-        $fulfillmentErrorState = $this->settingsRepository->getFulfillmentErrorState();
+        $states = [
+            'Fulfillment State'       => $this->settingsRepository->getFulfillmentState(),
+            'Pending State'           => $this->settingsRepository->getPendingState(),
+            'Fulfilled State'         => $this->settingsRepository->getFulfilledState(),
+            'Fulfillment Error State' => $this->settingsRepository->getFulfillmentErrorState(),
+        ];
+
+        foreach ($states as $k => $v) {
+            $this->checkIfFulfillmentSettingExists($v, $k);
+        }
 
-        $this->checkIfFulfillmentSettingExists($fulfillmentState, "Fulfillment State");
-        $this->checkIfFulfillmentSettingExists($fulfilledState, "Fulfilled State");
-        $this->checkIfFulfillmentSettingExists($fulfillmentErrorState, "Fulfillment Error State");
+        $duplicateStates = $this->keysOfDuplicateValues($states);
+        if ($duplicateStates === null) {
+            $this->showWordpressAdminErrorMessage("keysOfDuplicateValues: provided array must only have strings");
```

Comment:
has to be translated text with __()

Reply:
I can do it but it only serves developers valuable information. (this state only occurs if someone uses the function wrong)